File: apple-and-orange/appleorange.ts
const countApplesAndOranges = (s: number, t: number, a: number, b:number, apples: number[], oranges: number[]) => {
    let acount = 0
    let bcount = 0

    for (let i = 0; i < apples.length; i++) {
        let temp = a + apples[i]
        if (temp >= s && temp <= t) {
            acount++
        }
    }

    for (let i = 0; i < oranges.length; i++) {
        let temp =  + oranges[i]
        if (temp >= s && temp <= t) {
            bcount++
        }
    }

    console.log(acount)
    console.log(bcount)
}

File: apple-and-orange/appleorange.go
package main

func countApplesAndOranges(s, t, a, b int, apples, oranges []int) {
	acount := 0
	bcount := 0

	for _, apple := range apples {
		temp := a + apple
		if temp >= s && temp <= t {
			acount++
		}
	}

	for _, orange := range oranges {
		temp := b + orange
		if temp >= s && temp <= t {
			bcount++
		}
	}
}